embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/Embrace.java

```diff
@@ -484,41 +484,41 @@ public <T> T recordSpan(@NonNull String name, @Nullable EmbraceSpan parent, @Nul
     }
 
     @Override
-    public boolean recordCompletedSpan(@NonNull String name, long startTimeNanos, long endTimeNanos, @Nullable ErrorCode errorCode,
+    public boolean recordCompletedSpan(@NonNull String name, long startTimeMs, long endTimeMs, @Nullable ErrorCode errorCode,
```

Comment:
We should think how this change is communicated to customers - we'll definitely want to flag it in the changelog, and probably ask CS to update customers who are using these APIs

Reply:
Definitely. The backwards compat detection thing is a stopgap but current users, but I really want them to move away from this - fast.